kong/plugins/opentelemetry/otlp.lua

```diff
@@ -3,13 +3,14 @@ local pb = require "pb"
 local new_tab = require "table.new"
 local nkeys = require "table.nkeys"
 local tablepool = require "tablepool"
-local utils = require "kong.tools.utils"
+local cycle_aware_deep_copy = require("kong.tools.table").cycle_aware_deep_copy
+local kong_table = require "kong.tools.table"
```

Comment:
should we remove this line? And replace it by `local table_merge = require("kong.tools.table").table_merge `

Reply:
updated